react/javascript/tile-visualization/README.md

```diff
@@ -0,0 +1,184 @@
+
+   This is found under **Develop** => **Manage LookML Projects** => **New LookML Project**.
+
+   Select "Blank Project" as your "Starting Point". This will create a new project with no files.
+
+   1. The extension folder has a `manifest.lkml` file.
+
+   Either drag & upload this file into your Looker project, or create a `manifest.lkml` with the same content. Change the `id`, `label`, or `url` as needed.
+
+   ```
+   project_name: "tile-extension"
+
+   application: vis {
+      label: "Visualization Extension"
+      # file: "bundle.js"
+      url: "https://localhost:8080/bundle.js"
+      mount_points: {
+         dashboard_vis: yes
+         standalone: no
+      }
+      entitlements: {
+         local_storage: yes
+         use_form_submit: yes
+         core_api_methods: []
+         external_api_urls: []
+         oauth2_urls: []
+         scoped_user_attributes: []
+         global_user_attributes: []
+      }
+   }
+
+   application: tile {
+      label: "Tile Extension"
+      # file: "bundle.js"
+      url: "https://localhost:8080/bundle.js"
+      mount_points: {
+         dashboard_tile: yes
+         standalone: no
+      }
+      entitlements: {
+         local_storage: yes
+         use_form_submit: yes
+         core_api_methods: ["run_inline_query"]
+         external_api_urls: []
+         oauth2_urls: []
+         scoped_user_attributes: []
+         global_user_attributes: []
+      }
+   }
+   ```
+
+   Two separate extensions have been defined in the manifest above. This is not strictly necessary. Multiple mount points can be specified in the manifest. The following demonstrates how to define an extension that can be used for multiple mount points. Either way works just as well as the other. The decision to what definition to use is left to the extension developer and which is more convenient.
+
+   ```
+   project_name: "tile-extension"
+
+   application: app {
+      label: "Tile/Vizualization"
+      # file: "bundle.js"
+      url: "https://localhost:8080/bundle.js"
+      mount_points: {
+         dashboard_vis: yes
+         dashboard_tile: yes
+         standalone: yes
+      }
+      entitlements: {
+         local_storage: yes
+         use_form_submit: yes
+         core_api_methods: ["run_inline_query"]
+         external_api_urls: []
+         oauth2_urls: []
+         scoped_user_attributes: []
+         global_user_attributes: []
+      }
+   }
+
+   ```
+
+6. Create a `model` LookML file in your project. The name doesn't matter but the convention is to name it the same as the project— in this case, tile-visualization.
+
+- Add a connection in this model.
+- [Configure the model you created](https://docs.looker.com/data-modeling/getting-started/create-projects#configuring_a_model) so that it has access to the selected connection.
+  We do this because Looker permissions data access via models— In order to grant / limit access to an extension, it must be associated with a model.
```

Comment:
```suggestion
  We do this because Looker permissions data access via models — In order to grant / limit access to an extension, it must be associated with a model.
```